<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SEC</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Desktop/unnamed.jpg</normaloff>../../../Desktop/unnamed.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="commsBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Communications</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comPortComboBox">
        <item>
         <property name="text">
          <string>&lt;COM Port&gt;</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="connectButton">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="setUpBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>461</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Set Up</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>441</width>
       <height>142</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="syringeComboBox">
          <item>
           <property name="text">
            <string>25 mL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50 μL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>100 μL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>250 μL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>500 μL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1 mL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5 mL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10 mL</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="syringeButton">
          <property name="text">
           <string>Set Syringe Size</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="initializeButton">
        <property name="text">
         <string>Initialize Pump</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fillButton">
        <property name="text">
         <string>Fill Pump</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="emptyButton">
        <property name="text">
         <string>Empty Pump</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="emergencyStopBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>461</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Emergency Stop</string>
    </property>
    <widget class="QPushButton" name="stopButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>441</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>STOP PUMP</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="dispenseBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>461</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Dispense</string>
    </property>
    <widget class="QGroupBox" name="columnSelectBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>441</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>Column Selection</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>421</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="allCheckBox">
           <property name="text">
            <string>All</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QCheckBox" name="column1CheckBox">
             <property name="text">
              <string>Column 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="column2CheckBox">
             <property name="text">
              <string>Column 2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="column3CheckBox">
             <property name="text">
              <string>Column 3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="column4CheckBox">
             <property name="text">
              <string>Column 4</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QCheckBox" name="column5CheckBox">
             <property name="text">
              <string>Column 5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="column6CheckBox">
             <property name="text">
              <string>Column 6</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="column7CheckBox">
             <property name="text">
              <string>Column 7</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="column8CheckBox">
             <property name="text">
              <string>Column 8</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="dispenseButton">
         <property name="text">
          <string>Dispense to Columns</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="dispenseVolumeBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>441</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Dispense Volume</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>421</width>
        <height>33</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QDoubleSpinBox" name="dispenseSpinBox">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="value">
          <double>2.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dispenseUnits">
         <property name="text">
          <string>mL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dispenseVolumeButton">
         <property name="text">
          <string>Set Volume</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
